data/mil2525d/utilities/style-utilities/image-conversion-utilities/README.md

```diff
@@ -39,3 +39,20 @@
     *  Check the output for errors
 *  Check the destination folder for the converted image files
     * IMPORTANT NOTE: MS Paint (which is the default file association) does not support the latest version of EMF so you will need to use an alternative program to view the resulting EMF files (ex. MS Word)
+
+## Alternative Workflow using Adobe Illustrator Actions
+
+* If you have access to Adobe Illustrator (We are using CS6), you can use Actions in Batch Mode to convert your files from .svg to .emf.
+* IMPORTANT: This process will export .emfs to one singular folder, not the origin .svg file directory.
+* To start, download the `BatchConvertSVGtoEMF.aia` action set file contained in this repository.
```

Comment:
Providing direct link to this file would be convenient (but it is also in this folder)